src/Pivot.jl

```diff
@@ -0,0 +1,71 @@
+"""
+unstack_name(df::AbstractDataFrame, rowkeys, colkey, value)
+
+A helper function for unstack to name the columns being unstacked using the column name the values are present in.
+"""
+function unstack_name(df::AbstractDataFrame, rowkeys, colkey, value)
+    unstack(df::AbstractDataFrame, rowkeys, colkey, value, renamecols = x -> Symbol(string(value), "_", x))
+end
```

Comment:
Do we need this function? I see that it's only used once and is not exported. In that case, I would prefer the code to be in the in main pivot_wider function and just have brief comments describing what it does.

Reply:
I actually needed to broadcast this by accessing the value columns for the `renamecols` argument. However, I did not have enough knowledge to create a list of anonymous functions to broadcast here.  This should be probably simple without defining a function. I have now written an inline function inside the main function. I thought you said that it was gonna have performance issues, so I made it a separate function. So, please let me know if you know a way to do the broadcast without defining a function.